test/integration/utils/config_apply_utils.go

```diff
@@ -156,6 +171,34 @@ func CheckManifestFile(t *testing.T, container testcontainers.Container,
 	assert.Equal(t, expectedContent, manifestFiles)
 }
 
+func GetManifestFiles() map[string]*model.ManifestFile {
```

Comment:
```suggestion
func ManifestFiles() map[string]*model.ManifestFile {
```
Go convention doesn't like Get in function names